src/cms-content/region-overrides/region-overrides.json

```diff
@@ -75,11 +53,19 @@
       "context": "https://trello.com/c/7tqJyFna/1099-disabled-white-pine-county-nv-vaccine-data-due-to-anomoly"
     },
     {
-      "include": "region",
+      "include": "subregions",
       "metric": "metrics.caseDensity",
       "blocked": true,
-      "region": "47157",
-      "context": "https://trello.com/c/aB0EunUd/1108-disabled-shelby-county-tn-due-to-trailing-zeros-bringing-dnc-down"
+      "region": "OK",
```

Comment:
Worth adding a disclaimer here? Not sure if it shows up if it's fully blocked

Reply:
Yeah, it won't show up right now so there's no benefit, but that's a future improvement I've been thinking about...